### DISCUSSION TITLE
Equivalent of org.springframework.cloud.aws.autoconfigure.cache.ElastiCacheAutoConfiguration

### DISCUSSION THREAD

**Discussion by anekar422221** [awspring/spring-cloud-aws#1286] [Q&A]
Is there an equivalent of org.springframework.cloud.aws.autoconfigure.cache.ElastiCacheAutoConfiguration in this new spring-cloud-aws ?

**Comment by maciejwalkowiak** [comment]
There is no equivalent but if you're looking for integrating with ElastiCache Redis you can just use pure Spring + Redis integration.
